elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/query/SQLColumnProjection.java

```diff
@@ -111,7 +113,8 @@ default boolean canNest(Queryable source, MetaDataStore metaDataStore) {
 
         boolean requiresJoin = requiresJoin(references);
 
-        boolean inProjection = source.getColumnProjection(getAlias(), getArguments(), true) != null;
+        String columnId = (source instanceof SQLTable) ? getName() : getAlias();
```

Comment:
You can check if the source is the root instead of checking using the instance of operator here.